save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/basic/ProjectStatisticMenu.kt

```diff
@@ -32,24 +38,61 @@ external interface ProjectStatisticMenuProps : Props {
      * list of tests
      */
     var latestExecutionStatisticDtos: List<TestSuiteExecutionStatisticDto>?
+
+    /**
+     * Flag to open Menu
+     */
+    var isOpen: Boolean?
 }
 
 /**
+ * @param onOpenMenuStatistic
  * @return ReactElement
  */
 @Suppress("TOO_LONG_FUNCTION", "LongMethod", "MAGIC_NUMBER")
-fun projectStatisticMenu() =
+fun projectStatisticMenu(
+    onOpenMenuStatistic: (isOpen: Boolean) -> Unit,
+) =
         fc<ProjectStatisticMenuProps> { props ->
 
+            val (latestExecutionStatisticDtos, setLatestExecutionStatisticDtos) = useState(props.latestExecutionStatisticDtos)
+            val scope = CoroutineScope(Dispatchers.Default)
+
+            if (props.isOpen == true) {
+                useEffect(listOf<dynamic>(props.executionId, props.latestExecutionStatisticDtos)) {
```

Comment:
```suggestion
                useEffect(props.executionId, props.latestExecutionStatisticDtos) {
```
Actually, it seems to be a common error in our code - useEffect accepts a vararg, rather than a collection. Please check if it still works without `listOf`. We should fix other parts of code in the future too.

Reply:
Yes. It's work